src/js/product.ts

```diff
@@ -28,6 +29,36 @@ export default () => {
     const activeThumbnail = document.querySelector(SelectorsMap.product.activeThumbail(event.to));
 
     if (activeThumbnail) {
+      const activeThumbailRect = activeThumbnail.getBoundingClientRect();
```

Comment:
```suggestion
      const activeThumbnailRect = activeThumbnail.getBoundingClientRect();
```
This and all references below. IMO we should switch to term `thumb` instead of `thumbnail` just to shorten the code a bit.